scripts/gcs/a6000/photo_downloader.py

```diff
@@ -0,0 +1,21 @@
+import calendar
```

Comment:
We will need to redo this script with the proper way to get information from a request.